### ISSUE TITLE
Calendar links are missing a slash

### ISSUE THREAD

**Issue by melissawm** [scientific-python/scientific-python.org#199]
When clicking the "copy" button on the calendars page, the URL is missing a slash:

```
https:/scientific-python.org/calendars/numpy.ics
```

Thanks @sayantikabanik for noticing!

**Comment by stefanv** [comment]
I just ran into this myself, and I am very puzzled by it!  Here's the snippet of code that generates that link:

```
{{ $pageLink := .Permalink }}
{{ path.Join $pageLink (printf "%s.ics" $key) }}
```

Maybe some escaping.  I'll investigate and push a fix.

**Comment by stefanv** [comment]
Closed by 7e97bdccfd6c8d1ffb99b5ba8a5517256e792eec
